### PR TITLE
Fix broken routing

### PR DESCRIPTION
**Pull request by lockieRichter**
Fixes some broken routing and incorrect use of Riverpod.

Also fixes a bunch of minor issues.

### CODE REVIEW

File: lib/src/ui/boot_page.dart [hunk 1]
```diff
@@ -1,35 +1,11 @@
...
-
-class _BootPageState extends ConsumerState<BootPage> {
-  @override
-  void initState() {
-    unawaited(
-      ref.read(authenticationRepositoryProvider).getAccount().then((account) {
-        final String navigationUrl = account != null
-            ? DanteRoute.dashboard.navigationUrl
-            : DanteRoute.login.navigationUrl;
-        context.pushReplacement(navigationUrl);
```
Comment (shockbytes) [inline lib/src/ui/boot_page.dart:24]: Why did you remove it? Where are we checking if the customer is logged in?

Comment (lockieRichter) [reply lib/src/ui/boot_page.dart:24]: This is now handled in the app router provider instead.

Comment (lockieRichter) [reply lib/src/ui/boot_page.dart:24]: This also caused the app to crash sometimes at start up

**Review by shockbytes — APPROVED** [review]
Nice!
